plugins/catalog-backend-module-bitbucket-server/api-report.md

```diff
@@ -0,0 +1,97 @@
+## API Report File for "@backstage/plugin-catalog-backend-module-bitbucket-server"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BitbucketServerIntegrationConfig } from '@backstage/integration';
+import { Config } from '@backstage/config';
+import { Entity } from '@backstage/catalog-model';
+import { EntityProvider } from '@backstage/plugin-catalog-backend';
+import { EntityProviderConnection } from '@backstage/plugin-catalog-backend';
+import { LocationSpec } from '@backstage/plugin-catalog-backend';
+import { Logger } from 'winston';
+import { TaskRunner } from '@backstage/backend-tasks';
+
+// @public (undocumented)
+export type BitbucketProject = {
+  key: string;
+};
+
+// @public (undocumented)
+export type BitbucketRepository = {
+  project: {
+    key: string;
+  };
+  slug: string;
+  description: string;
+  links: Record<
+    string,
+    {
+      href: string;
+    }[]
+  >;
+};
+
+// @public
+export class BitbucketServerClient {
+  constructor(options: { config: BitbucketServerIntegrationConfig });
+  // (undocumented)
+  static fromConfig(options: {
+    config: BitbucketServerIntegrationConfig;
+  }): BitbucketServerClient;
+  // (undocumented)
+  getRepository(projectKey: string, repo: string): Promise<BitbucketRepository>;
```

Comment:
I feel like we should be wrapping up these arguments in order to make changes to the API in a non-breaking fashion in the future without adding more and more arguments.

`getRepository(opts: { projectKey: string, repo: string })`

There's a few of them in this `api-report` that could do with wrapping up 👍